frictionless/checks/cell/ascii_value.py

```diff
@@ -0,0 +1,70 @@
+from ... import errors
+from ...check import Check
+from typing import Union, Iterator
+
+
+class ascii_value(Check):
+    """Check for forbidden values in a field
+
+    API      | Usage
+    -------- | --------
+    Public   | `from frictionless import checks`
+    Implicit | `validate(checks=[{"code": "ascii-value"}])`
+
+    This check can be enabled using the `checks` parameter
+    for the `validate` function.
+
+    Parameters:
+       forbidden_values? (any[]): a list of forbidden values
+
+    """
+
+    code = "ascii-value"
+    Errors = [errors.NotAscii]
+
+    def __init__(
+        self, descriptor: Union[list, None] = None, *, forbidden_values: list = []
```

Comment:
I would remove `forbidden_values` option as we already have `checks.forbidden_value`